script/import_scripts/bbpress.rb

```diff
@@ -84,7 +84,7 @@ def import_users
           id: u["id"].to_i,
           username: u["user_nicename"],
           email: u["user_email"].downcase,
-          name: u["display_name"],
+          name: u["display_name"].length > 0 ? u["display_name"] : u['user_nicename'],
```

Comment:
Fancy. 

OK. I did that and committed again. Is that what I was supposed to do?